website/docs/r/license.html.markdown

```diff
@@ -0,0 +1,25 @@
+---
+layout: "nsxt"
+page_title: "NSXT: nsxt_license"
+sidebar_current: "docs-nsxt-resource-license"
+description: |-
+  Provides a resource to configure license on NSX-T manager
+---
+
+# nsxt_license
+
+Provides a resource to configure license on NSX-T manager
+
+## Example Usage
+
+```hcl
+resource "nsxt_license" "license" {
+  license_key = "0000-00000-00000-00000-00000"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `license_key` - (Required) NSX-T license key.
```

Comment:
Hi Coleman, looks good, just one nit - if import is supported, it should be mentioned here in the docs.
I also think that resources that are dependent on this license will fail to create/delete together with the licence, unless an explicit depends_on is specified in resource config. I don't think we can fix this easily, but worth mentioning this here in the doc.